CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## [Unreleased]
 
-- No changes yet.
+- Add `--exit code` flag to `buf format` to exit with a non-zero exit code if
+  the files were not already formatted.
```

Comment:
(you may mean --exit-code not --exit code)

Reply:
lol, yes -- we were a bit excited about this. Thanks for catching this and we'll update the changelog :)